src/Serilog.Sinks.AzureDataExplorer/AzureDataExplorerSinkOptions.cs

```diff
@@ -78,30 +120,43 @@ public int QueueSizeLimit
         public string ApplicationKey { get; private set; }
         public X509Certificate2 ApplicationCertificate { get; private set; }
         public string Authority { get; private set; }
-        public bool SendX5c { get; private set; }
+        public bool SendX5C { get; private set; }
         public string AzureRegion { get; private set; }
         public Azure.Core.TokenCredential TokenCredential { get; private set; }
 
         public AzureDataExplorerSinkOptions()
         {
-            Period = TimeSpan.FromSeconds(10);
-            BatchPostingLimit = 1000;
-            QueueSizeLimit = 100000;
+            this.Period = TimeSpan.FromSeconds(10);
+            this.BatchPostingLimit = 1000;
+            this.QueueSizeLimit = 100000;
+            this.BufferFileRollingInterval = RollingInterval.Hour;
+            this.BufferFileCountLimit = 31;
+            this.BufferFileSizeLimitBytes = 100L * 1024 * 1024;
```

Comment:
31 .. Is that monthly ? Do we want to reduce that ?
100MB file ?

Reply:
reduced the BufferFileCountLimit to 20 and BufferFileSizeLimitBytes to 10MB.